### ISSUE TITLE
Assignment 2

### ISSUE THREAD

**Issue by talond** [IanSan/CSC415_OSPrinciples#5]
## New system calls
- [x] kill: stop a process
- [x] remove: remove a file or directory
- [x] chdir: changes the current working directory of the calling process to the directory specified
## New processes
- [x] kill: stop a process; uses kill
- [x] rm: remove file; uses remove
- [x] cp: copy file; uses open, read, write, close
- [x] ps: list processes
- [x] ls: list files in directory
- [x] cat: print a file to standard output
- [x] more: prints one screenful of text at a time
## Shell commands
- [x] cd: change shell working directory; uses chdir

**Comment by mariorsjr** [comment]
Can we do the remove putting a flag instead of deleting the file, right?
I mean, to show files the flag deleted must be false. When remove we can put the flag true and don't show when ls is called (linux uses this)

**Comment by talond** [comment]
Yeah, we can do that. We still need to figure out a filesystem structure so we track those things.

**Comment by mariorsjr** [comment]
A tree?
example (imagine a folder named Folder)
<img width="188" alt="screen shot 2016-04-01 at 10 54 04 am" src="https://cloud.githubusercontent.com/assets/2224461/14215470/171bda66-f7f8-11e5-8b90-9f5a5e16a594.png">

**Comment by betaedwin** [comment]
I'm on my lunch at work. Il go over the thread when I'm off work.

On Friday, April 1, 2016, Mario Rodrigues notifications@github.com wrote:

> A tree?
> example (imagine a folder named Folder)
> [image: screen shot 2016-04-01 at 10 54 04 am]
> https://cloud.githubusercontent.com/assets/2224461/14215470/171bda66-f7f8-11e5-8b90-9f5a5e16a594.png
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> https://github.com/IanSan/CSC415_OSPrinciples/issues/5#issuecomment-204489986

## 

Edwin Betancourt

San Francisco State University | Computer Engineering

(408)882-7698 | betaedwin@gmaill.com

**Comment by IanSan** [comment]
ps is mine -- Ian S

**Comment by JerryG05** [comment]
Working on ls

**Comment by betaedwin** [comment]
Il work on more

**Comment by mariorsjr** [comment]
I will work on rm
